tests/Test.js

```diff
@@ -24,6 +24,7 @@ const HMICapabilities = SDL.rpc.structs.HMICapabilities;
 const WindowState = SDL.rpc.structs.WindowState;
 const Grid = SDL.rpc.structs.Grid;
 const WindowStatus = SDL.rpc.structs.WindowStatus;
+const Temperature = SDL.rpc.structs.Temperature;
 
```

Comment:
Can you setup the `ClimateData` struct?

on line 28, you can add `const ClimateData = SDL.rpc.structs.ClimateData;`

Reply:
Fixed in d1e6617